runtime/src/eval/system.rs

```diff
@@ -268,30 +268,10 @@ pub fn create<H: Handler>(
 
 	match handler.create(runtime.context.address, scheme, value, &code, None) {
 		Capture::Exit((reason, address, return_data)) => {
-			runtime.return_data_buffer = return_data;
-			let create_address: H256 = address.map(|a| a.into()).unwrap_or_default();
-
-			match reason {
-				ExitReason::Succeed(_) => {
-					push!(runtime, create_address.into());
-					Control::Continue
-				},
-				ExitReason::Revert(_) => {
-					push!(runtime, H256::default());
-					Control::Continue
-				},
-				ExitReason::Error(_) => {
-					push!(runtime, H256::default());
-					Control::Continue
-				},
-				ExitReason::Fatal(e) => {
-					push!(runtime, H256::default());
-					Control::Exit(e.into())
-				},
-			}
+			save_created_address(runtime, reason, address, return_data, handler)
 		},
 		Capture::Trap(interrupt) => {
-			push!(runtime, H256::default());
+			// push!(runtime, H256::default());
```

Comment:
Write a short description of why you commented out this line.

Reply:
This case was not implemented.
Now the case is used to implement the "create" opcode